pkg/controller/storagecluster/update.go

```diff
@@ -104,6 +115,15 @@ func (c *Controller) rollingUpdate(cluster *corev1.StorageCluster, hash string)
 				numUnavailableKvdb++
 			}
 		}
+
+		// If pxversion is greater than 2.15 then delete the portworx-api pods with csi-node-registrar containers as well when updating storage cluster
```

Comment:
*2.13

If you are adding a comment, please explain why it needs to be deleted. Because we want to re-register the csi driver which gets removed when the registrar is removed from the px pod.